docs/content/getting-started/perform-check.mdx

```diff
@@ -140,13 +141,13 @@ To obtain the [access token](https://auth0.com/docs/get-started/authentication-a
 </TabItem>
 </Tabs>
 
-### 02. Calling check API
+### 02. Calling Check API
 
-To check whether user `user:anne` has relationship `reader` with object `document:Z`
+To check whether user `user:anne` has relationship `can_view` with object `document:Z`
```

Comment:
[The model](https://openfga.dev/pr-preview/pr-878/docs/getting-started/configure-model) doesn't have the `can_view` relation (it has `reader`, `writer`, `owner`). Do we want to retain `reader` or add `can_view` and `can_edit` to that model?

Reply:
Didn't realize that all those pages were assuming the same model was being used. I'll update to follow the model.